src/plugins/recordTypes/person/forms/default.jsx

```diff
@@ -100,6 +100,109 @@ const template = (configContext) => {
       <Panel name="hierarchy" collapsible collapsed>
         <Field name="relation-list-item" subpath="rel:relations-common-list" />
       </Panel>
+
+      <Panel name="supplied" collapsible collapsed>
```

Comment:
This panel should go above Hierarchy. I'd like to keep Hierarchy consistently at the bottom.